art/paths.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+CHECKPOINTS_PATH = Path("checkpoints")
+LOG_PATH = Path("logs")
+EXPERIMENT_LOG_PATH = CHECKPOINTS_PATH/"experiment"/"logs"
```

Comment:
The structure is okay but root folder should be user defined. What if I want to run 2 or more experiments in one folder. All will be in the same place?

I mean we have this `exp1` defined in our first tutorial

Reply:
than it will be run in another folder so it will by in exp2